org.eclipse.scanning.api/src/org/eclipse/scanning/api/scan/event/IPositioner.java

```diff
@@ -78,7 +78,7 @@
 	 * @return monitors
 	 * @throws ScanningException
 	 */
-	List<IScannable<?>> getMonitors()  throws ScanningException;
+	List<IScannable<?>> getMonitorsPerPoint()  throws ScanningException;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)